tests/FSharp.Compiler.ComponentTests/CompilerService/AsyncMemoize.fs

```diff
@@ -211,6 +211,44 @@ let ``Job is restarted if first requestor cancels but keeps running if second re
 type ExpectedException() =
     inherit Exception()
 
+[<Fact>]
```

Comment:
Maybe this should go to `FSharp.Compiler.UnitTests/BuildGraphTests.fs` ?

Reply:
Yep! I'll take care of that.